.aspect/workflows/config.yaml

```diff
@@ -5,312 +5,122 @@ workspaces:
         label: rules_js
     e2e/bzlmod:
         icon: bazel
-        tasks:
+        tasks: &e2e_tasks
             - test:
                   queue: aspect-medium
             - format:
                   without: true
             - buildifier:
                   without: true
+            - bazel-8:
```

Comment:
@alexeagle doesn't this have a side-effect on every test?

Reply:
What side-effect do you mean?

Reply:
I don't know, but removing this diff implies bazel-8 is no longer tested? I don't understand how that is related to "remove old aspect CLI"?

Reply:
Oh I see, thanks 👍 

So we'll start enabling multiple bazel versions in workflows now?